python/desc/imsim/atmPSF.py

```diff
@@ -200,4 +228,9 @@ def _getPSF(self, xPupil=None, yPupil=None, gsparams=None):
             t0=self.t0,
             exptime=self.exptime,
             gsparams=gsparams)
+        if self.gaussianFWHM > 0.0:
+            psf = galsim.Convolve(
```

Comment:
@jmeyers314 @rmjarvis Unfortunately, doing this convolution breaks this code:  https://github.com/lsst/sims_GalSimInterface/blob/master/python/lsst/sims/GalSimInterface/galSimInterpreter.py#L1006
Could you guys have a look please?

Reply:
Ah, right.  I forgot we got rid of a feature we used to have where if any argument to a Convolve was itself a Convolution, it would combine the lists.  I guess we could do that manually in `PSFbase.applyPSF`:
```
if isinstance(psf, galsim.Convolution):
    return galsim.Convolution([obj] + psf.obj_list)
else:
    return galsim.Convolution(obj, psf)
```

Reply:
Thanks, making that change and swapping the order of the items in the `Convolve` in `atmPSF._getPSF` addresses the issue.   I'll go ahead and make the PR in sims_GalSimInterface.